### PR TITLE
librsvg: fix fetch, configure, and build (#18865)

### PR DESCRIPTION
**Pull request by omsai**

### CODE REVIEW

File: var/spack/repos/builtin/packages/librsvg/package.py [hunk 1]
```diff
@@ -12,21 +12,22 @@ class Librsvg(AutotoolsPackage):
     homepage = "https://wiki.gnome.org/Projects/LibRsvg"
     url      = "https://download.gnome.org/sources/librsvg/2.44/librsvg-2.44.14.tar.xz"
 
-    version('2.44.14', sha256='6a85a7868639cdd4aa064245cc8e9d864dad8b8e9a4a8031bb09a4796bc4e303')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/librsvg/package.py:15]: Can we keep the old version?

**Review by alalazo — CHANGES_REQUESTED** [review]
Just a minor request, otherwise LGTM

File: var/spack/repos/builtin/packages/librsvg/package.py (hunk 1 above)
Comment (omsai) [reply var/spack/repos/builtin/packages/librsvg/package.py:15]: The old version didn't compile for me.  Does it still work for you?

Comment (alalazo) [reply var/spack/repos/builtin/packages/librsvg/package.py:15]: I didn't try to build it, but usually we don't remove old versions of libraries when new ones are available. We should start doing that at some point, but policies are still under discussion #15896

**Comment by omsai** [comment]
@alalazo I understand.  I've restored the older package version.

Also I took the liberty to rebase and squashed my 3 commits, because they were adding noise and I don't recall ever seeing my previous PRs being merge squashed.

I don't know why GitHub decided to throw away my message in the comment thread explaining these things when I hit "Resolve" but perhaps it had something to do with the rebase.

**Review by alalazo — APPROVED** [review]

**Pull request merged**